[2017-12-22 16:37:31.690] [DEBUG   ] --- START computation for graph 20 5 5 (__main__:57)
[2017-12-22 16:37:31.690] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-22 16:37:31.690] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-22 16:37:31.697] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
[2017-12-23 02:37:31.690] [ERROR   ] --- Timed out (mru.computevalue:137)
Traceback (most recent call last):
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 62, in compute_values
    csr = compute_covering_routes(graph, tgts, rm_dominated=rm_dominated)
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 215, in compute_covering_routes
    covset = cs.computeCovSet(graph_game, ver, targets)
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 73, in computeCovSet
    C = purgeDominatedStrategies(C, i+1);
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 97, in purgeDominatedStrategies
    condition2 = np.array_equal(np.sort(routes[n]), np.sort(routes[m])); # we want that the routes contains the same elements
  File "/usr/lib/python2.7/dist-packages/numpy/core/fromnumeric.py", line 824, in sort
    a = asanyarray(a).copy(order="K")
  File "/home/multidefender/mru_securitygame/mru/patrolling/correlated/correlated_row_gen.py", line 29, in signal_handler
    raise TimeoutException, "Timed out"
TimeoutException: Timed out
[2017-12-23 02:37:31.697] [DEBUG   ] --- saved results in /home/multidefender/mru_securitygame/mru/../file/results/res_graphs_20_ntgts/res_instance_ntgts_20_den_25_dead_5_graphix_5_ix_0.pickle (mru.iomanager:121)
[2017-12-23 02:37:31.697] [DEBUG   ] --- END computation for graph 20 5 5 (__main__:64)
[2017-12-23 02:37:32.475] [DEBUG   ] --- START computation for graph 20 15 1 (__main__:57)
[2017-12-23 02:37:32.475] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-23 02:37:32.476] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-23 02:37:32.484] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
[2017-12-23 12:37:32.476] [ERROR   ] --- Timed out (mru.computevalue:137)
Traceback (most recent call last):
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 62, in compute_values
    csr = compute_covering_routes(graph, tgts, rm_dominated=rm_dominated)
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 215, in compute_covering_routes
    covset = cs.computeCovSet(graph_game, ver, targets)
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 73, in computeCovSet
    C = purgeDominatedStrategies(C, i+1);
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 96, in purgeDominatedStrategies
    condition1 = len(routes[n]==i); # we just deal with the last layer of routes, i.e. the longest ones
  File "/home/multidefender/mru_securitygame/mru/patrolling/correlated/correlated_row_gen.py", line 29, in signal_handler
    raise TimeoutException, "Timed out"
TimeoutException: Timed out
[2017-12-23 12:37:32.502] [DEBUG   ] --- saved results in /home/multidefender/mru_securitygame/mru/../file/results/res_graphs_20_ntgts/res_instance_ntgts_20_den_25_dead_15_graphix_1_ix_0.pickle (mru.iomanager:121)
[2017-12-23 12:37:32.502] [DEBUG   ] --- END computation for graph 20 15 1 (__main__:64)
[2017-12-23 12:37:33.446] [DEBUG   ] --- START computation for graph 20 19 7 (__main__:57)
[2017-12-23 12:37:33.446] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-23 12:37:33.447] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-23 12:37:33.454] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
[2017-12-23 22:37:33.447] [ERROR   ] --- Timed out (mru.computevalue:137)
Traceback (most recent call last):
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 62, in compute_values
    csr = compute_covering_routes(graph, tgts, rm_dominated=rm_dominated)
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 215, in compute_covering_routes
    covset = cs.computeCovSet(graph_game, ver, targets)
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 73, in computeCovSet
    C = purgeDominatedStrategies(C, i+1);
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 98, in purgeDominatedStrategies
    condition3 =  m not in deleted and n not in deleted;
  File "/home/multidefender/mru_securitygame/mru/patrolling/correlated/correlated_row_gen.py", line 29, in signal_handler
    raise TimeoutException, "Timed out"
TimeoutException: Timed out
[2017-12-23 22:37:33.467] [DEBUG   ] --- saved results in /home/multidefender/mru_securitygame/mru/../file/results/res_graphs_20_ntgts/res_instance_ntgts_20_den_25_dead_19_graphix_7_ix_0.pickle (mru.iomanager:121)
[2017-12-23 22:37:33.467] [DEBUG   ] --- END computation for graph 20 19 7 (__main__:64)
[2017-12-23 22:37:33.985] [DEBUG   ] --- START computation for graph 25 10 3 (__main__:57)
[2017-12-23 22:37:33.985] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-23 22:37:33.986] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-23 22:37:34.  1] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
[2017-12-24 08:37:33.986] [ERROR   ] --- Timed out (mru.computevalue:137)
Traceback (most recent call last):
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 62, in compute_values
    csr = compute_covering_routes(graph, tgts, rm_dominated=rm_dominated)
  File "/home/multidefender/mru_securitygame/mru/computevalue.py", line 215, in compute_covering_routes
    covset = cs.computeCovSet(graph_game, ver, targets)
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 73, in computeCovSet
    C = purgeDominatedStrategies(C, i+1);
  File "/home/multidefender/mru_securitygame/mru/srg/computecovsets.py", line 98, in purgeDominatedStrategies
    condition3 =  m not in deleted and n not in deleted;
  File "/home/multidefender/mru_securitygame/mru/patrolling/correlated/correlated_row_gen.py", line 29, in signal_handler
    raise TimeoutException, "Timed out"
TimeoutException: Timed out
[2017-12-24 08:37:34.  0] [DEBUG   ] --- saved results in /home/multidefender/mru_securitygame/mru/../file/results/res_graphs_25_ntgts/res_instance_ntgts_25_den_25_dead_10_graphix_3_ix_0.pickle (mru.iomanager:121)
[2017-12-24 08:37:34.  0] [DEBUG   ] --- END computation for graph 25 10 3 (__main__:64)
[2017-12-24 08:37:34.508] [DEBUG   ] --- START computation for graph 30 5 9 (__main__:57)
[2017-12-24 08:37:34.508] [DEBUG   ] --- start compute_values function (mru.computevalue:44)
[2017-12-24 08:37:34.508] [DEBUG   ] --- compute shortest sets (mru.computevalue:56)
[2017-12-24 08:37:34.521] [DEBUG   ] --- compute covering routes (mru.computevalue:60)
